src/Serializer/AbstractItemNormalizer.php

```diff
@@ -305,7 +305,16 @@ protected function getAttributeValue($object, $attribute, $format = null, array
     {
         $propertyMetadata = $this->propertyMetadataFactory->create($context['resource_class'], $attribute, $this->getFactoryOptions($context));
 
-        $attributeValue = $this->propertyAccessor->getValue($object, $attribute);
+        try {
+            $attributeValue = $this->propertyAccessor->getValue($object, $attribute);
+        } catch(NoSuchPropertyException $e) {
+            if (null === $propertyMetadata->getInheritance()) {
+                throw $e;
+            }
+
+            $attributeValue = null;
```

Comment:
I'm not very fond of this but it's a required test for Doctrine inherited properties. For example, property `origin` can be available on `EntityFirst` but not on `EntitySecond`, even though they all extend the abstract `EntityAbstract`.